pkg/apps/installer.go

```diff
@@ -334,6 +334,17 @@ func (i *Installer) update() error {
 		}
 	}
 
+	oldTermsVersion := oldManifest.Terms().Version
+	newTermsVersion := newManifest.Terms().Version
+
+	termsAdded := oldTermsVersion == "" && newTermsVersion != ""
+	termsUpdated := oldTermsVersion != newTermsVersion
```

Comment:
Just to be sure, we agree that `termsAdded` implies `termsUpdated`? Ie, when the `oldTermsVersion` is empty, and `newTermsVersion` is not, they can't be equal, and thus `termsUpdated` is true.

Reply:
Yes, this is exactly that